crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -203,6 +206,16 @@ where
         })
     }
 
+    fn get_total_pieces(&self) -> RpcResult<u64> {
```

Comment:
Shouldn't this be in farmer metadata? We already have a few fields there of a similar nature.

Reply:
I thought at first about that, but metadata contains actual metadata, which rarely change (only with runtime), while total pieces counter is contained inside node storage

Reply:
Yes, but it is not actually guaranteed that max plot size, for instance, will never change. We also only really need this during initial sync, so we query it just once anyway.